test/regress/regress0/preprocess/circuit-prop.smt2

```diff
@@ -0,0 +1,53 @@
+; COMMAND-LINE: --proof-new-req --check-proofs-new
```

Comment:
```suggestion
; COMMAND-LINE:
```

Reply:
The first option won't ever be put in master and the second will eventually be built in to run_regression.py.